qa/setup_dali_extra.sh

```diff
@@ -1,13 +1,24 @@
 #!/bin/bash
 
 # Fetch test data
-export DALI_EXTRA_PATH=/opt/dali_extra
+
+# User can specify optional argument with target where to download DALI_extra
+if [ "$#" -eq 1 ]; then
+    export DALI_EXTRA_PATH=$1
+else
+    export DALI_EXTRA_PATH=/opt/dali_extra
+fi
+
 DALI_EXTRA_URL="https://github.com/NVIDIA/DALI_extra.git"
+DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
+DALI_EXTRA_TAG_PATH="${DIR}/../DALI_EXTRA_TAG"
```

Comment:
You call it DALI_EXTRA_**TAG**, but in reality it's a hash. Maybe use some more generic wording that does not imply a particular Git construct - e.g. DALI_EXTRA_VERSION / DALI_EXTRA_VER.

Reply:
Done